### PR TITLE
Prepare for build and shipit config to publish package

### PR DESCRIPTION
**Pull request by samrose3**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,7 @@ yarn-error.log*
 # Shopify
 /.shadowenv.d/
 /.dev/
+
+# yalc
+.yalc
+yalc.lock
```
Comment (alex-page) [inline .gitignore:24]: We can probably exclude this as it's not what will happen long term.

File: src/@polaris/components/src/components/Text/Text.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export interface TextProps
 
 export const useTextStyles = ({
   weight = 'regular',
-  type = 'body',
+  type = 'system',
```
Comment (alex-page) [inline src/@polaris/components/src/components/Text/Text.tsx:30]: Happy to leave this here but could be another PR

**Review by alex-page — APPROVED** [review]

File: src/@polaris/components/src/components/Text/Text.tsx (hunk 2 above)
Comment (samrose3) [reply src/@polaris/components/src/components/Text/Text.tsx:30]: Yeah, not the best PR/git hygine here mixing scopes. It was needed to fix TS build errors

File: .gitignore (hunk 1 above)
Comment (samrose3) [reply .gitignore]: Removed 💥

File: package.json [hunk 3]
```diff
@@ -1,9 +1,20 @@
 {
-  "name": "polaris",
-  "version": "0.0.0",
-  "private": true,
+  "name": "@shopify/layout-experimental",
```
[the remark this answers is not in the document]
Comment (samrose3) [reply package.json:2]: Good idea 👍 

#163
